component/main.jsonnet

```diff
@@ -66,12 +68,34 @@ local readServices = kube.ClusterRole('appcat:services:read') + {
   ],
 };
 
-{
-  '10_clusterrole_view': xrdBrowseRole,
-  [if isOpenshift then '10_clusterrole_finalizer']: finalizerRole,
-  '10_clusterrole_services_read': readServices,
+// adding namespace for syn-appcat
+local ns = kube.Namespace('syn-appcat');
+// adding missing namespace for slos
+local slos_namespace = kube.Namespace('appcat-slos');
+
+// add secret
+
+local secret = kube.Secret('mailgun-smtp-credentials') {
+  metadata+: {
+    namespace: 'syn-appcat',
+  },
+  stringData: {
+    password: pgParams.mailgun_smtp_password,
```

Comment:
There doesn't seem to be a vault reference in the `defaults.yaml` where does this come from?

Reply:
That;s good hint, I'll move it a little bit up in hierarchy - I wanted to add valut values to syn, because each cluster has it's own secret - should it be rather here?

Reply:
If you put the values into the vault you still need a reference in the component.

You can look at other vault values in the component to see how they are handled. For example this here: https://github.com/vshn/component-appcat/blob/master/class/defaults.yml#L125

Also add some documentation how we get those secrets, so everyone could install AppCat on a new cluster.